<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RA_MainWindow_User</class>
 <widget class="QMainWindow" name="RA_MainWindow_User">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>699</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>49</y>
      <width>201</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_1_1">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_1_2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="label_1_3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>201</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2_1">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>300</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2_2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>320</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2_3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>50</y>
      <width>201</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3_1">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_8">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>300</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_8">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3_2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_9">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>320</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_9">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3_3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_10">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>50</y>
      <width>201</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_10">
     <item row="0" column="0">
      <widget class="QLabel" name="label_4_1">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_11">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>300</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_11">
     <item row="0" column="0">
      <widget class="QLabel" name="label_4_2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_12">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>320</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_12">
     <item row="0" column="0">
      <widget class="QLabel" name="label_4_3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_13">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>201</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_13">
     <item row="0" column="0">
      <widget class="QLabel" name="label_5_1">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_14">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>610</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_14">
     <item row="1" column="0">
      <widget class="QLabel" name="label_5_2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_15">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>630</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_15">
     <item row="0" column="0">
      <widget class="QLabel" name="label_5_3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_16">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>360</y>
      <width>201</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_16">
     <item row="0" column="0">
      <widget class="QLabel" name="label_6_1">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_17">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>610</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_17">
     <item row="0" column="0">
      <widget class="QLabel" name="label_6_2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_18">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>630</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_18">
     <item row="0" column="0">
      <widget class="QLabel" name="label_6_3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_19">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>360</y>
      <width>201</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_19">
     <item row="0" column="0">
      <widget class="QLabel" name="label_7_1">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_20">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>610</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_20">
     <item row="0" column="0">
      <widget class="QLabel" name="label_7_2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_21">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>630</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_21">
     <item row="0" column="0">
      <widget class="QLabel" name="label_7_3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_22">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>360</y>
      <width>201</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_22">
     <item row="0" column="0">
      <widget class="QLabel" name="label_8_1">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_23">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>610</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_23">
     <item row="0" column="0">
      <widget class="QLabel" name="label_8_2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_24">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>630</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_24">
     <item row="0" column="0">
      <widget class="QLabel" name="label_8_3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>10</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>查看详情</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionShare"/>
   <addaction name="actionReview"/>
   <addaction name="actionAddActivity"/>
   <addaction name="actionActivity"/>
   <addaction name="actionUserCenter"/>
  </widget>
  <action name="actionShare">
   <property name="text">
    <string>分享书籍</string>
   </property>
  </action>
  <action name="actionReview">
   <property name="text">
    <string>发布书评</string>
   </property>
  </action>
  <action name="actionAddActivity">
   <property name="text">
    <string>发起活动</string>
   </property>
  </action>
  <action name="actionActivity">
   <property name="text">
    <string>查看活动</string>
   </property>
  </action>
  <action name="actionUserCenter">
   <property name="text">
    <string>个人中心</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
